src/containerapp/HISTORY.rst

```diff
@@ -4,6 +4,7 @@ Release History
 ===============
 upcoming
 ++++++
+* 'az containerapp sessionpool create': fix default container name to be lower case letters
```

Comment:
* 'az containerapp sessionpool create': Fix default container name to be lower case letters